test/suite/test_backup14.py

```diff
@@ -330,23 +286,23 @@ def insert_bulk_data(self):
         self.add_data(self.uri_logged, 'bulk')
         self.take_full_backup()
         self.take_incr_backup()
-        self.compare_backups(self.uri_logged)
-
+        self.compare_backups(self.uri_logged, self.home_full, self.home_incr, str(self.counter))
+        self.compare_backups(self.uri, self.home_full, self.home_incr, str(self.counter))
```

Comment:
This line here is the only one that is a functionality change. You modified the line that compared `uri_logged` and then added this new comparison. Was that intentional? It was unexpected to me.

Reply:
oops sorry, accidently added this line. Thanks!